### PR TITLE
Fixed uninitialized memory read in FloatComplexVectorList

### PR DESCRIPTION
**Pull request by sebastianbaginski**
Fixes https://github.com/cogneco/ooc-kean/issues/952
@marcusnaslund

### CODE REVIEW

File: test/math/FloatComplexVectorListTest.ooc [hunk 1]
```diff
@@ -34,7 +34,7 @@ FloatComplexVectorListTest: class extends Fixture {
 	complexNumberArray add(complexNumber2)
 	complexNumberArray add(complexNumber3)
 
-	tolerance := 0.00001
+	tolerance := 0.00001f
```
Comment (marcusnaslund) [inline test/math/FloatComplexVectorListTest.ooc]: Aaaaaaah. I've fallen for this so many times. Nice catch.

Comment (sebastianbaginski) [inline test/math/FloatComplexVectorListTest.ooc]: Looks like Constraints can't handle different parameter types for tolerance and float arguments.
This change fixed the uninitialized memory read.

File: test/math/FloatComplexVectorListTest.ooc [hunk 2]
```diff
@@ -70,10 +70,10 @@ FloatComplexVectorListTest: class extends Fixture {
...
 			sum := list sum
 			mean := list mean
-			expect(sum real == 6.0f)
-			expect(sum imaginary == 4.0f)
-			expect(mean real == 1.5f)
-			expect(mean imaginary == 1.0f) //FIXME getting errors using "is equal to" here because... ooc (expected 1.00000000 got 1.00000000)
+			expect(sum real, is equal to(6.0f) within(tolerance))
+			expect(sum imaginary, is equal to(4.0f) within(tolerance))
+			expect(mean real, is equal to(1.5f) within(tolerance))
+			expect(mean imaginary, is equal to(1.0f) within(tolerance))
```
Comment (sebastianbaginski) [inline test/math/FloatComplexVectorListTest.ooc]: These lines are only to get rid of TODO.

File: test/math/FloatComplexVectorListTest.ooc (hunk 1 above)
Comment (marcusnaslund) [inline test/math/FloatComplexVectorListTest.ooc]: That's strange since IIRC they're all cast to `LDouble` anyway.

Comment (sebastianbaginski) [inline test/math/FloatComplexVectorListTest.ooc]: That's right. I don't feel like digging deeper into this though :)

**Comment by marcusnaslund** [comment]
Tests now work on Windows as well. :)

**Pull request merged**